apps/frontend/src/app/5_pages/App/App.tsx

```diff
@@ -20,7 +21,7 @@ function App() {
     useWalletConnect();
 
   return (
-    <>
+    <div className="relative">
```

Comment:
This breaks the footer sticking to bottom of screen, and positioning of Osano cookie consent button

![image](https://user-images.githubusercontent.com/765825/202141568-b7ec0c8f-d549-49b1-bf88-b552b0dbee1b.png)